ui/app/src/components/AuditGrid/styles.ts

```diff
@@ -24,6 +24,8 @@ export const useStyles = makeStyles()((theme) => ({
     }
   },
   cellRoot: {
+    display: 'flex',
```

Comment:
Added the styles here since that's already being used all over the Data Grid columns definition https://github.com/craftercms/studio-ui/blob/develop/ui/app/src/components/AuditGrid/AuditGridUI.tsx#L192, adding sx would imply more updates (thinking about the release)